project/src/behaviors/GSTFace.gd

```diff
@@ -0,0 +1,26 @@
+extends GSTMatchOrientation
+class_name GSTFace
+"""
+Calculates angular acceleration to rotate a target to face its target's position.
+The acceleration will attempt to arrive with zero remaining angular velocity.
+"""
+
+
+func _init(agent: GSTSteeringAgent, target: GSTAgentLocation).(agent, target) -> void:
+	pass
+
+
+func _face(acceleration: GSTTargetAcceleration, target_position: Vector3) -> GSTTargetAcceleration:
+	var to_target: = target_position - agent.position
+	var distance_squared: = to_target.length_squared()
```

Comment:
I know it's a very minor thing, but let's try and use the same convention everywhere. In `Utils.clampv3()` we have `var len2: = vector.length_squared()`, here we have `var distance_squared: = to_target.length_squared()`. Let's pick either `*_squared` or `*2` and stick with it through the codebase

Reply:
In that case, I'd go with _squared - that's how they do it in the Vector class.